### ISSUE TITLE
Manejo de errores con archivos incorrectos

### ISSUE THREAD

**Issue by teofum** [PI-ITBA/2024_01#441]
Es necesario manejar errores de acceso a memoria en la lectura de archivo en el caso que el programa se ejecute con los argumentos incorrectos? Por ejemplo, los nombres de agencia de NYC son más cortos que los de Chicago, con lo cual si se ejecuta el programa `parkingTicketsNYC` con el dataset `ticketsCHI.csv` incorrectamente, al leer el archivo va a causar un error de acceso a memoria y abortar. Deberíamos manejar este caso?

No me queda claro porque por un lado, el enunciado indica que el programa nunca debería abortar, pero por otro lado también indica que se puede asumir que los datos son correctos, lo cual incluye el largo de los campos, y no es el caso si se llama con el archivo/dataset incorrecto.

Gracias!

**Comment by teofum** [comment]
Claro, me refería al caso de ejecutar el programa con el dataset incorrecto (por ejemplo, el programa compilado para NYC con el dataset de Chicago)

**Comment by marcelogarberoglio** [comment]
Que el formato no es el que tiene que ser sino el de otra ciudad es un caso puntual de "el formato no es el correcto".
